src/Containers/WaitTimeDisplay.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { flex } from '@Utils/Mixins';
+import { SmallText } from "@Text";
+import styled, {useTheme} from "styled-components";
+
+const BUSY = 30;
+const STARTING_TO_GET_BUSY = 15;
+
+export interface IWaitTimeDisplay {
+    /**
+     * The seating/reservation length of average wait time
+     */
+    AverageWaitTime: number,
+}
+
+/**
+ * Primary UI component for user interaction
+ * WaitTimeDisplay
+ */
+export const WaitTimeDisplay: React.FC<IWaitTimeDisplay>
+    = ({
+           AverageWaitTime = 29,
+            ...props
+       }) => {
+    return (
+        <Row {...props}>
+                <PieChart AverageWaitTime={AverageWaitTime} />
+            <TextPaddingLeftTop>
+                    <SmallText size={'3rem'} bold>Avg. Wait</SmallText><br />
+                    <SmallText size={'3rem'} bold >{AverageWaitTime} Min</SmallText>
+            </TextPaddingLeftTop>
+        </Row>
+    );
+};
+
+/**
+ * This function will determine what color the Pie Chart will be
+ * @param AverageWaitTime {Number} - Number given from getTimeDifference() function
+ * @return {string} - Hexadecimal of color
+ */
+function getColor(AverageWaitTime: Number){
+    const {colors} = useTheme();
+    if(AverageWaitTime >= BUSY){
+        return colors.PieChartColors.Red;
+    } else if(AverageWaitTime >= STARTING_TO_GET_BUSY && AverageWaitTime <= BUSY){
+        return 	colors.PieChartColors.Yellow;
+    } else {
+        return colors.PieChartColors.Green;
+    }
+}
```

Comment:
This entire thing should be refactored into the Styled Component Location since this logic is assiociated with a specific component not the entire thing?

Reply:
Should I refractor this this into the PieChart component or move the function right above the component?

Reply:
I would move it next to the styled component becuase  it only relates to that component